lib/config.ts

```diff
@@ -16,11 +17,11 @@ export async function getConfig(): Promise<CrawlConfig> {
 
   const userConfig: CrawlConfig = await prompt([
     {
-      type: "input",
+      type: "select",
       name: "gitTag",
       message: `What git tag do you want to crawl?`,
-      initial: process.argv.slice(2)[0] || "master",
```

Comment:
If I remember correctly, this still needs to be here.
Otherwise when skip this question (by running `npx find-deprecations master`), the `gitTag` will be undefined

Reply:
done